### PR TITLE
Added wish-list page

### PR DESCRIPTION
**Pull request by jeetsdev**
## Wishlist Page

- Wishlist items
  - Remove from wishlist
  - Move to cart

image

### CODE REVIEW

File: css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+/** Wishlist products section style here */
+.wishlist__product-sec {
+    width: 80%;
+}
+
+/** Wishlist items cards style here  */
+.wishlist__product-item .card__sec-lower {
+    padding: 0;
+}
```
Comment (pritam-kr) [inline css/wishlist.css]: remove these red mark by breaking line

**Review by pritam-kr — APPROVED** [review]
All looks good to merge

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wishlist-SudoMart</title>
+    <link rel="stylesheet" href="../css/wishlist.css">
+</head>
```
Comment (Tarandeep-s1ngh) [inline pages/wishlist.html:9]: Better to use your own component library for cards and other components

**Review by Tarandeep-s1ngh — APPROVED** [review]

**Review by ikkurthis1998 — APPROVED** [review]
I can see that you are not using any components from your component library. My suggestion is if your component library doesn't have any component, once you make the component here, add it to component library as well. This way you can get a richer component library by the end of the camp

File: pages/wishlist.html (hunk 2 above)
Comment (jeetsdev) [reply pages/wishlist.html:9]: Actually, I am importing my component lib in my common.css file and importing that common file in the entire project.

**Comment by jeetsdev** [comment]
> I can see that you are not using any components from your component library. My suggestion is if your component library doesn't have any component, once you make the component here, add it to component library as well. This way you can get a richer component library by the end of the camp

Actually, I am importing my component lib in my common.css file and importing that common file in the entire project.

**Pull request merged**